testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/authentication/FlowTest.java

```diff
@@ -315,6 +325,17 @@ public void addExecutionFlow() {
 
         authMgmtResource.addExecutionFlow("parent", params);
         assertAdminEvents.assertEvent(testRealmId, OperationType.CREATE, AdminEventPaths.authAddExecutionFlowPath("parent"), params, ResourceType.AUTH_EXECUTION_FLOW);
+
+        // add execution flow with alias that contains reserved characters
+        params.put("alias", "Child/of:browser");
+        try {
+            authMgmtResource.addExecutionFlow("parent", params);
```

Comment:
Wouldn't it be better to use some form of `assertThrows(ClientErrorException.class, ...) `?

Reply:
Thanks for your review @Delawen 
That would have been my first choice too, but I wanted to keep in-line with the way `ClientErrorException`s are handled in existing test cases, e.g.
https://github.com/keycloak/keycloak/blob/6391651768def2e018de309fd02ffcbc426282e8/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/authentication/FlowTest.java#L378-L384

I changed the tests to use `assertThrows` instead.